src/sidebar/components/PendingUpdatesNotification.tsx

```diff
@@ -55,13 +55,16 @@ function PendingUpdatesNotification({
   }
 
   return (
-    <div role="status" className="animate-fade-in">
+    <div
+      role="status"
+      className="absolute right-0 animate-updates-notification-slide-in"
+    >
       <Button
         onClick={applyPendingUpdates}
         unstyled
         classes={classnames(
           'flex gap-1.5 items-center py-1 px-2',
-          'rounded shadow-lg bg-gray-900 text-white',
+          'rounded shadow-lg bg-gray-900 text-white whitespace-nowrap',
```

Comment:
What happens without this?

Reply:
It looks like this 

![image](https://github.com/hypothesis/client/assets/2719332/f2e2e883-104d-4e62-b033-d9599d47595a)

I suppose it is because the container is now `position: absolute`. Perhaps we could try setting the container with `w-full`. I can try that.

Reply:
> I suppose it is because the container is now position: absolute.

You need to figure out how the sizing of the element is determined when `whitespace-nowrap` is missing.